src/scripts/clientUtils.js

```diff
@@ -19,12 +19,6 @@ export async function serverAddress() {
         return Promise.resolve(apiClient.serverAddress());
     }
 
-    const current = await ServerConnections.getAvailableServers().then(servers => {
-        if (servers.length !== 0) {
-            return Promise.resolve(servers[0].ManualAddress);
-        }
-    });
-
     // TODO this makes things faster but it also blocks the wizard in some scenarios
     // if (current) return Promise.resolve(current);
 
```

Comment:
I still think comment should be removed because the associated code is removed in this PR.
```suggestion
```
_Hope this removes 3 lines - GH shows only 2_

Reply:
Opened a separate PR since it is not _strictly_ related to fixing the warnings here. :wink:

https://github.com/jellyfin/jellyfin-web/pull/2559